.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] [OPTIONAL] You should implement the above requirements in all repositories in your pair-programming group, kindly try to implement this in future projects